Add devel doc partially (#4)
SUSE/sap_host_exporter: doc/devel.md

> # Developing

Reviewer: following up on my previous suggestion, here is a more clear title:

```suggestion
# Developer notes
```

> # Learning materials:

Reviewer: maybe use a lower heading level
```suggestion
## Learning materials
```

This guide will provide some guided to understand the material behind Netweaver.
The link might be broken or some small changes will needed during the time, just submit a PR to fix if you find broken links.

> This guide goal is to provide some entrypoints.

Reviewer: typos and wording
```suggestion
This guide will provide some initial pointers to understand the material behind Netweaver.
The link might be broken or some small changes will be needed in the future; please feel free to submit a PR to fix any broken links you find.
```

> ##  Exploratory/Learning SOAP of SapControl interface:

Reviewer: wording
```suggestion
##  Exploring and Learning the SOAP SapControl interface
```

You can find the full documentation here: https://www.sap.com/documents/2016/09/0a40e60d-8b7c-0010-82c7-eda71af511fa.html?infl=547aee22-5559-4c30-9b86-3e2df9e5d806

> In order to learn the interface of SOAP you can use the folowing sapcontrol script: ( example extracted from previous link and adapted)

Reviewer: typos and wording
```suggestion
In order to learn the SOAP interface, you can use the following Python script: (example extracted from the previous link and adapted)
```
